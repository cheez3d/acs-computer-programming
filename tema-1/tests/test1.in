Afara ninge
144244402228801777726644412
iarna e anotimpul alb
2
iarna toamna
alb multicolor
Hai afara la zapada
