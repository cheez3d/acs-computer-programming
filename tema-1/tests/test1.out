123332777206644466433
Hai cu SaniA
866626#6620330266#6668444678855506885558444222666555666777
4
1442444 233327772 5552 999927232 
999927232 4
0
99 92 77 72 55 52 44 42 40 33 32 27 23 20 14 9 
